simulator/examples/analog-clock.rs

```diff
@@ -155,33 +140,32 @@ fn main() {
         .build_binary();
 
     loop {
-        display.clear();
-
-        display.draw(draw_face());
-
         let time = Local::now();
 
-        display.draw(draw_hour_hand(time.hour()));
-        display.draw(draw_minute_hand(time.minute()));
-        display.draw(draw_seconds_hand(time.second()));
-
         // NOTE: In no-std environments, consider using
         // [arrayvec](https://stackoverflow.com/a/39491059/383609) and a fixed size buffer
-        let clock_text = format!(
+        let digital_clock_text = format!(
             "{:02}:{:02}:{:02}",
             time.hour(),
             time.minute(),
             time.second()
         );
 
+        display.clear();
+
+        display.draw(draw_face());
+        display.draw(draw_hour_hand(time.hour()));
+        display.draw(draw_minute_hand(time.minute()));
+        display.draw(draw_seconds_hand(time.second()));
+
         // Draw digital clock just above center
-        display.draw(draw_digital_clock(&clock_text));
+        display.draw(draw_digital_clock(&digital_clock_text));
 
         // Draw a small circle over the hands in the center of the clock face. This has to happen
         // after the hands are drawn so they're covered up
         display.draw(
-            Circle::new(CENTER, 6)
-                .fill(Some(BinaryColor::Off))
+            Circle::new(CENTER, 4)
+                .fill(Some(BinaryColor::On))
                 .stroke(Some(BinaryColor::On)),
```

Comment:
The stroke isn't normally necessary if the stroke and fill colors are equal, but #143 currently breaks circles without stroke. Should we remove the stroke here and accept the small visual error until #143 is fixed?

Reply:
> Should we remove the stroke here and accept the small visual error until #143 is fixed?

Ah true, yeah makes sense. I've pushed a change that just removes the `.stroke()` call.